docs/v2/11_web3wallet-mobile-and-desktop.md

```diff
@@ -172,39 +173,43 @@ public class Web3WalletSignVerify : MonoBehaviour
 
 ### Hosting Your Own Sign Page {#hosting-your-own-sign-page}
 
-You can find the Web3Wallet repo here [https://github.com/ChainSafe/game-web3wallet](https://github.com/ChainSafe/game-web3wallet) you can download and host this on git for free. This instantiates web3 in webpage to allow MetaMask to connect & make transactions. We'll guide you through this process below.
+You can find the Web3Wallet repo [here](https://github.com/ChainSafe/game-web3wallet). You can download and host this on git for free. This instantiates web3 in a webpage to allow MetaMask to connect & make transactions. We'll guide you through this process below.
 
-Step 1: Create a GitHub account [https://github.com/join](https://github.com/join)\
+Step 1: [Create a GitHub account](https://github.com/join).
 
-
-Step 2: Clone the web3wallet repo using GitHub Desktop
+Step 2: Clone the [Web3Wallet](https://github.com/ChainSafe/game-web3wallet) repo using GitHub Desktop. Press `Add`, `Clone Repository...`, and then paste in the following URL: `https://github.com/ChainSafe/game-web3wallet`.
```

Comment:
We might be able to get away with just saying "Clone the repo" and not need to point them to use Github Desktop; in fact, I think we don't want to advertise GD usage bc that is kind of potato compared to command line (and will poorly reflect) unless we have previously said somewhere that GD is to be used?

Reply:
Mention of GH Desktop is specifically what Sneakz had written in the first iteration of the v2 docs AFAICT. We can consider rewriting this in a refactor.